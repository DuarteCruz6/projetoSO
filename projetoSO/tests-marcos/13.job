WRITE [(x2,74)(i6,34)(y5,3)(p8,61)]
WRITE [(a6,33)(p6,1)(n3,74)(i2,51)]
WAIT 2000
READ [n3]
WRITE [(b1,69)(a2,64)]
DELETE [a2,i6]
WRITE [(t3,10)]
WRITE [(v8,3)(r3,97)(s2,75)(j0,14)(p5,82)]
WRITE [(n1,100)(m2,85)(g2,36)(o8,40)(u8,92)]
READ [x2]
WRITE [(e2,23)(j1,98)(j8,23)]
WRITE [(j2,28)(z3,89)]
WRITE [(e3,63)]
WRITE [(x1,60)(k6,100)(v0,56)(s6,27)]
WRITE [(t9,28)(u7,20)(w8,42)(f5,54)(t7,26)]
READ [b1]
DELETE [m2,o8,p6,y5,z3]
WRITE [(p6,30)(v6,86)(k5,15)]
SHOW
READ [n3]
WAIT 1000
DELETE [k6]
BACKUP
WRITE [(g9,75)(k2,50)]
WRITE [(v9,2)(x9,59)(c3,17)(a5,36)]
DELETE [i2,x2,b1,j2,g2]
READ [v0]
WRITE [(e8,35)(l6,55)(a4,52)]
READ [l6]
WRITE [(q4,2)]
DELETE [n1,c3]
WRITE [(c5,88)(g3,15)(b2,42)(x6,1)]
WRITE [(p7,58)(j6,12)]
WRITE [(l7,43)(m8,48)]
WRITE [(z7,93)(o4,52)(i7,97)(q6,24)]
WRITE [(s9,3)(h6,52)]
