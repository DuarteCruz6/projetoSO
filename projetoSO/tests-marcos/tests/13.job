WRITE [(u7,6)(p8,50)(d5,70)(e6,77)]
WRITE [(u4,32)(x3,27)]
WRITE [(f4,65)(u1,98)(x8,32)]
WRITE [(m0,71)(u2,57)(t2,36)(c8,36)]
SHOW
WAIT 1000
READ [u1]
BACKUP
READ [x3]
WAIT 3000
WRITE [(m5,21)(e9,11)]
WRITE [(n0,51)(d9,54)(w2,28)(s1,99)(p9,84)]
DELETE [u2]
DELETE [f4,d9,p9,x8]
WAIT 2000
DELETE [c8]
DELETE [m0,d5]
WAIT 1000
WRITE [(g0,37)(y6,89)(t1,67)]
WRITE [(m0,43)]
WRITE [(j7,86)(z2,95)(f8,31)]
WRITE [(x9,8)(e7,4)(i1,0)(i9,0)]
DELETE [n0,u7,x3]
WAIT 3000
SHOW
WRITE [(c9,23)(x6,24)(z4,45)(m4,20)(z3,100)]
WAIT 2000
WRITE [(c4,2)]
WRITE [(r7,82)(n2,48)]
READ [c9]
WRITE [(o3,9)(o2,95)(n0,81)(h6,92)(d6,67)]
WRITE [(q2,50)]
DELETE [u4,x6,m0]
READ [z4]
WRITE [(p2,12)]
DELETE [c4,i1,s1]
READ [e6]
SHOW
READ [w2]
WRITE [(c6,22)]
READ [t1]
WRITE [(x2,46)(u8,97)(q1,68)]
WRITE [(l4,98)]
WAIT 1000
BACKUP
WRITE [(d0,74)(b6,73)(z6,49)(n3,54)(j1,6)]
WRITE [(f2,72)]
DELETE [z3,x9,r7]
