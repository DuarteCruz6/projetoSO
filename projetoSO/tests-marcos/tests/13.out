(c8, 36)
(d5, 70)
(e6, 77)
(f4, 65)
(m0, 71)
(p8, 50)
(t2, 36)
(u2, 57)
(u1, 98)
(u4, 32)
(u7, 6)
(x8, 32)
(x3, 27)
[(u1,98)]
[(x3,27)]
(e7, 4)
(e9, 11)
(e6, 77)
(f8, 31)
(g0, 37)
(i9, 0)
(i1, 0)
(j7, 86)
(m0, 43)
(m5, 21)
(p8, 50)
(s1, 99)
(t1, 67)
(t2, 36)
(u1, 98)
(u4, 32)
(w2, 28)
(x9, 8)
(y6, 89)
(z2, 95)
[(c9,23)]
[(z4,45)]
[(e6,77)]
(c9, 23)
(d6, 67)
(e7, 4)
(e9, 11)
(e6, 77)
(f8, 31)
(g0, 37)
(h6, 92)
(i9, 0)
(j7, 86)
(m4, 20)
(m5, 21)
(n0, 81)
(n2, 48)
(o2, 95)
(o3, 9)
(p2, 12)
(p8, 50)
(q2, 50)
(r7, 82)
(t1, 67)
(t2, 36)
(u1, 98)
(w2, 28)
(x9, 8)
(y6, 89)
(z3, 100)
(z4, 45)
(z2, 95)
[(w2,28)]
[(t1,67)]
